code_examples/sdk_examples/src/core_features/getting_started/03_fetch_did.ts

```diff
@@ -1,16 +1,16 @@
 /* eslint-disable prefer-const */
 import * as Kilt from '@kiltprotocol/sdk-js'
+import * as Did from '@kiltprotocol/did'
```

Comment:
I am fairly certain this doesn't exist… DID-related methods are part of the `Kilt.DidHelpers.` and ``Kilt.DidResolver.`.

Or is this some other way to access the methods @rflechtner ?

Reply:
Unfortunately, I couldn’t find another way to access the `Did.linkedInfoFromChain` function. I found `apiConfig.call.did.linkedInfoFromChain(encodedKiltnerd123Details)`, but it doesn’t seem to work the same way and gives errors.

Reply:
we are indeed missing a function to resolve a DID document based on a w3n. This could be something worth adding to the DidResolver that Chris mentioned